test/plugins/features/movie/controller.test.js

```diff
@@ -13,12 +13,49 @@ describe('movie controller', () => {
       return Controller.create(payload)
       .then((movie) => {
         expect(movie.get('title')).to.eql(payload.title);
+      });
+    });
 
-        return new Movie({ id: movie.id }).fetch();
-      })
-      .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-        expect(movie.get('release_year')).to.eql(payload.release_year);
+  });
+
+  describe('get', () => {
+
+    const movie = { title: 'foo', release_year: 2016 };
+
+    beforeEach(() => {
+      return Knex('movies').insert(movie);
+    });
+
+    it('gets a movie', () => {
+      const query = {};
+
+      return Controller.list(query)
+      .then((movies) => {
+        expect(movies.length).to.be.above(0);
```

Comment:
Nit: There's actually a Chai helper to assert length: `expect(movies).to.have.length(1);`.

In addition, it's usually better to assert an exact value as opposed to a general assertion (greater than 0). That way your expectations are very explicit, and you're making sure exactly what you expect comes back.